### ISSUE TITLE
Discussion : go up / down hierarchy keybinding clashes with existing mental model of text navigation.

### ISSUE THREAD

**Issue by hikchoi** [dendronhq/dendron#481]
**Is your feature request related to a problem? Please describe.**
the default keybinding for `goDownHierarchy` is <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Down</kbd>. 

I don't know about other people but this in my head is _jump to end_ (<kbd>Cmd</kbd> + <kbd>Down</kbd>) while _selecting everything_ (+ <kbd>Shift</kbd>).

Same goes for <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Up</kbd> for `goUpHierarchy`.

**Describe the solution you'd like**
I'm not sure, but it should be a key combination that isn't commonly used in text manipulation / navigation.

**Describe alternatives you've considered**
1. Adapt myself to use <kbd>Shift</kbd> + <kbd>Start</kbd> / <kbd>End</kbd> for multi line selection
2. remap default go up/down hierarchy to something that doesn't clash with my mental model.

**Additional context**
I'm kinda torn, because having it mapped like this really does make sense for going up/down a hierarchy, yet it clashes with an existing mental model of navigating a text file.

This is easily fixed on my end by just remapping keys, but I thought it was worth a discussion as I'm sure I'm not the only one in the world using these keys for selecting multiple lines of code, and a simple keymapping like this could be something that introduce resistance to newcomers and frustrate them.

**Comment by kevins8** [comment]
Keyboard shortcuts are tricky. I'm basically trying to find something that
doesn't conflict with what people already have and makes sense for the
context.
Part of me is thinking to switch to a chorded approach for Dendron
shortcuts (eg. CMD+G DOWN) which will make shortcuts more semantic and less
likely to conflict

**Comment by hikchoi** [comment]
yeah shortcuts are a bit of an headache. more so because we are building on top of an existing software that already has a bunch of shortcuts itself 😄 

Also, seems habit is something that's hard to force away. I heard from someone at Adobe Research saying they have such an hard time introducing shortcuts / hotkeys for new features or deprecating old keybindings to make room for new ones because there will always be people who would complain and that's not acceptable for them.

Chorded shortcuts could be a nice way to handle it (<kbd>Cmd</kbd> + <kbd>G</kbd> is a bit of a hard maneuver though lol). I haven't thought of the UI/UX of keyboard shortcuts from a developer's perspective until now and being mostly a user of a software, I normally just remapped something that clashed with my normal workflow so that I use all three modifier keys + whatever the original alphabet was bound to. If that maneuver was a bit too hard, I would go on and flash my keyboard with a custom keybinding on a separate layer.

Slightly off topic, but I think it's worth surveying other knowledge management systems and their shortcuts and see if theres anything worth adopting. I see a lot of folks coming from other systems and having a familiar keyboard shortcut would make the switch easier (not sure if that goes against fair competition 😇 ). Of course, with go up/down command, it's something that is unique to Dendron so this will not apply.

Or maybe an exhaustive survey of common shortcuts on different levels: OS / (text entry) applications / common plugins / knowledge management applications. (too brute-forcey?)

**Comment by aleksey-rowan** [comment]
Backlining to a discussion this issue has started:
- https://github.com/dendronhq/dendron/discussions/700
